packages/routing-interface/util/RIFUtil.ts

```diff
@@ -43,7 +43,7 @@ export class RIFUtil {
             return JSON.parse(jsonStr);
         }
         catch (error) {
-            logger.error(error)
+            // logger.error(error)
```

Comment:
@takeutak Is this change intended to be in #444 or #442 ?

cc: @sfuji822 @hartm @jonathan-m-hamilton

Reply:
Clarified over the maintainer's call: #442

Reply:
Thank you for your comment.  As your comment on PR #442, I corrected this file.
https://github.com/hyperledger/cactus/pull/444/commits/b2d1f476f626122cdeaaa68aaeb08099fa496e41